### DISCUSSION TITLE
Is there a way to create only the types?

### DISCUSSION THREAD

**Discussion by raky291** [openapi-ts/openapi-typescript#2008] [Q&A]
I would like to know if it is possible to create only the types (Interfaces, Enums) and preserve the origial names from OpenAPI, is that possible?

I know it creates types in the following way:

`type MyType = components["schemas"]["MyType"];`

But I'm looking for it to create types directly (also enums) and preserve the original name.

**Comment by nahasco** [comment]
I realised this has been supported for a long time after the merge of this PR #1876 
Add this flag to the command `--root-types --root-types-no-schema-prefix`
